### DISCUSSION TITLE
Why do we compare screenshots after the completion of test?

### DISCUSSION THREAD

**Discussion by pftg** [donv/capybara-screenshot-diff#59] [Q&A] [answered]
Hey @donv / @UweKubosch , could you share why the comparison of the screenshot is processing on the teardown stage?

**Comment by donv** [comment, marked as answer]
1. To me the graphical comparison is not very interesting if the functional part of the test fails.
2. The graphical comparison is pretty resource intensive, so if it can be skipped, the tests run faster.
